protobufs/livekit_models.proto

```diff
@@ -248,15 +250,35 @@ message UserPacket {
   repeated string destination_identities = 6 [deprecated=true];
   // topic under which the message was published
   optional string topic = 4;
+  // Unique ID to indentify the message
+  optional string id = 8;
+  // start and end time allow relating the message to specific media time
+  optional uint64 start_time = 9;
+  optional uint64 end_time = 10;
 
-  // NEXT_ID: 7
+  // NEXT_ID: 11
 }
 
 message SipDTMF {
   uint32 code = 3;
   string digit = 4;
 }
 
+message Transcription {
+  string participant_identity = 2;
```

Comment:
This entry was taken from the initial proposal. Is it needed considering that the top level `DataPacket` message provide a `participant_identity` field as well?

Reply:
yeah, I think it's needed within as well. in DataPacket, `participant_identity` represents the user that sent it. here it represents the user who's transcription we are providing. For example, if a backend agent is performing transcriptions, the outer envelope would be the Agent's identity, and here it would be the track publisher's.